front/src/utils/cast-as-positive-integer-or-null.ts

```diff
@@ -8,7 +8,7 @@ export function canBeCastAsPositiveIntegerOrNull(
   if (typeof probablePositiveNumberOrNull === 'number') {
     return (
       Number.isInteger(probablePositiveNumberOrNull) &&
-      Math.sign(probablePositiveNumberOrNull) === 1
+      Math.sign(probablePositiveNumberOrNull) !== -1
```

Comment:
I think we should also update the tests!

Reply:
Done !